module3/lessons/Week2/RestfulRoutes.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title:  RESTful Routes
+---
+
+## Learning Objectives
+*
+*
+
```

Comment:
I think it could be helpful to do a review of CRUD here  - that way you can associate the routes with different crud actions.  Since we won't yet have a fully formed MVC app, students can still see how the different HTTP Methods match up to our standard Database actions.